sound/soc/sof/amd/acp-ipc.c

```diff
@@ -148,23 +148,38 @@ static void acp_dsp_ipc_get_reply(struct snd_sof_dev *sdev)
 irqreturn_t acp_sof_ipc_irq_thread(int irq, void *context)
 {
 	struct snd_sof_dev *sdev = context;
+	const struct sof_amd_acp_desc *desc = get_chip_info(sdev->pdata);
```

Comment:
commit message: I am not sure what "until unless" could possibly mean.

Also please add quotes 
Now we deassert the interrupt at top half which solves the below
"nobody cared" warning.

Reply:
Ok